Dockerfile

```diff
@@ -27,15 +27,14 @@ RUN apk add postgresql-client && \
 
 RUN apk add python3-dev
 
-RUN apk add --no-cache python3 py3-pip && \
+RUN apk add --no-cache python3 py3-pip & \
  if [ ! -e /usr/bin/pip ]; then ln -s pip3 /usr/bin/pip ; fi && \
  if [[ ! -e /usr/bin/python ]]; then ln -sf /usr/bin/python3 /usr/bin/python; fi
 
 
-RUN apk add build-base libffi-dev libressl-dev && \
+RUN apk add build-base libffi-dev && \
     ln -sf /usr/bin/python3 /usr/bin/python && \
```

Comment:
Missing a slash at the start of one of the ln's but also isn't these done above? (Tho the one above could use a full path just so it's clear and less error-prone.

I realise you didn't change this, I just noticed. I'll make an issue and you can do now or someone do later.

Reply:
Good spot, I see there's an issue now at https://github.com/IATI/IATI-Standard-Website/issues/771